scripts/anaconda-xorg-shim

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+
+# Xorg sends SIGUSR1, but we use that to force-run exception handler that terminates anaconda.
+# If we give up waiting on Xorg, and it still manages to start eventually, it kills us this way.
+# Hence this shim to translate the signal to something else.
+
+import signal
+import subprocess
+import sys
+import os
+
+
+def handler(num, frame):
+    message = "Xorg signal shim received SIGUSR1 from Xorg, passing on as SIGRTMIN+8."
+    subprocess.run(["systemd-cat"], input=message.encode("ascii"))
+    os.kill(os.getppid(), signal.SIGRTMIN + 8)
+
+
+def preexec():
+    signal.signal(signal.SIGUSR1, signal.SIG_IGN)
+
+
+message = "Xorg signal shim started by {} as {}.".format(os.getppid(), os.getpid())
+subprocess.run(["systemd-cat"], input=message.encode("ascii"))
+params = ["/bin/Xorg"] + sys.argv[1:]
+old_handler = signal.signal(signal.SIGUSR1, handler)
+process = subprocess.Popen(params, preexec_fn=preexec)
+process.wait()
+signal.signal(signal.SIGUSR1, old_handler)
```

Comment:
I'm not sure if keeping the handler is the right thing to do in the shim script. The alternative would be `SIG_IGN`.